packages/data-context/src/sources/VersionsDataSource.ts

```diff
@@ -29,16 +29,15 @@ export class VersionsDataSource {
   constructor (private ctx: DataContext) {
     this._initialLaunch = true
     this._currentTestingType = this.ctx.coreData.currentTestingType
-    this.#ensureData()
```

Comment:
If we call this when the data source is initialised (eg when the entire lifecycle starts) the config has not been read yet, and we cannot get the `config.component` key. So, we don't call this immediately anymore, it's called just-in-time by `versionData`.